apps/examples/wc-vanilla/main.js

```diff
@@ -5,6 +5,7 @@ document.querySelector('#app').innerHTML = `
     <div>
         <h1>Hello World!</h1>
         <pie-button id="counter" type="button"></pie-button>
+        <pie-icon-button size="medium" type="button" variant="primary"></pie-icon-button>
```

Comment:
Is there deliberately no icon provided here?

Reply:
@ashleynolan resolving this to get the code in because it shouldn't block but would be good to know